examples/http-server-oak-crud-middleware-with-sqlite3-db.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * @title HTTP server: Performing CRUD operations using SQLite3
+ * @difficulty intermediate
+ * @tags web, cli, deploy
+ * @run -A <url>
+ * @group Network
+ *
+ * An example of a HTTP server for CRUD routes with oak middleware framework and SQLite3 database.It demonstrates the CRUD(Create, Read, Update and Delete) operations on file-based SQLite Database using HTTP methods (Get, Post, Put, Delete, Options)
+ */
```

Comment:
Add used resources here, so that it's easier for users to know what's being used and what to look up for more information.